components/shared/theme.js

```diff
@@ -22,46 +22,58 @@ const breakpoints = {
 
```

Comment:
Why do you have white and black in all palette colors?
Does not sound weird to have a white danger color, or a secondary black color?

And, I see that you are not using the commitizen `(yarn commitizen)` to manage your commits, It would be nice to use it, the semantic-release read them and put in the release notes based on the conventional commit pattern.

https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines

Reply:
I inserted this colors in all palette just to prevent issues if a different theme try to use the 0 or 1000 color.

@allyssonsantos do you think it would be better to do otherwise and remove the 0 and 1000 from remainings palletes ?

And thanks to warning me to use the `yarn commitizen`.